widget/playground/src/components/SideNavigation/SideNavigation.styles.ts

```diff
@@ -32,6 +32,7 @@ export const Tab = styled(Flex, {
   padding: '$15 0',
   cursor: 'pointer',
   width: '100%',
+  height: '105px',
```

Comment:
it's defined twice. one here and another one on `widget/playground/src/components/SideNavigation/SideNavigation.Tabs.tsx` and `TAB_HEIGHT`.

Make them using same value.

Reply:
Thanks, Fixed.